README.md

```diff
@@ -12,6 +12,95 @@ If you create a connected app using below link, you need not pass client Id and
+#### 3. Create pkcs8 private key from the key pair
+```
+openssl pkcs8 -topk8 -nocrypt -in keypair.key -out private.key
+```
+
+![Terminal key generation screenshot](images/terminal-key-generation.png)
+> This creates keypair.key, private.key and certificate.crt files in that folder. Files private.key and certificate.crt will be used later in the setup.
+### Connected App Setup
+
+This flow utilizes Salesforce's server to server [JWT bearer flow](https://help.salesforce.com/articleView?id=remoteaccess_oauth_jwt_flow.htm&type=5) for acquiring an access token. This portion of the setup will walk you through setting up the connected app.
+
+1. Login to Salesforce → Setup and Search "**App Manager**"
+2. In the Setup’s Quick Find search "**App Manager**"
+3. Select **“New Connected App”**
+    1. Connected App Name: CDP API
+    2. API Name: CDP_API (or whatever default value is pre-populated)
+    3. Contact Email: Your email address
+    4. Under API Heading, check the box for **“Enable OAuth Settings”**
+    5. Callback URL: https://oauth.pstmn.io/v1/callback
+    6. Select the checkbox for **“Use digital signatures”**
+    7. Select **“Choose File”** and select the **certificate.crt** file created in [Create a Private Public Key Pair and a Public Digital Certificate](#Create a Private Public Key Pair and a Public Digital Certificate) section
+    8. Under **“Selected OAuth Scopes”** move the following from the “Available OAuth Scopes” to “Selected OAuth Scopes”
+        1. Manage user data via APIs (api)
+        2. Perform requests at any time (refresh_token, offline_access)
+        3. Perform ANSI SQL queries on Salesforce CDP data (cdp_query_api)
+        4. Manage Salesforce CDP profile data (cdp_profile_api)
+        5. Manage Salesforce CDP Ingestion API data (cdp_ingest_api)
+        6. Note: feel free to select others if needed.
+
+       > Your screen should look similar to this
+
+       ![Connected App Setup 1 Screenshot](images/connected-app-setup-1.png)
+    9. Select **Save** (on the next screen select **Continue**)
+    10. Make note of the **Consumer Key** value. This will be used as the **“clientId”** variable for establishing connection.
+
+        ![Connected App Setup 2 Screenshot](images/connected-app-setup-2.png)
+
+4. At the top of your newly created connected app click “Manage”
+    1. Select **“Edit Policies”**
+    2. Change **“IP Relaxation”** to **“Relax IP restrictions”**
+    3. Select **Save**
+
+## App Authorization
+
+At this point your connected app has been configured however there is a ***_one time setup requirement_*** to authorize your user with the connected app.
+
+The URL format will look like:
+
+```
+<YOUR_ORG_URL>/services/oauth2/authorize?response_type=code&client_id=<YOUR_CONSUMER_KEY>&scope=api refresh_token cdp_profile_api cdp_query_api cdp_ingest_api&redirect_uri=https://oauth.pstmn.io/v1/callback
```

Comment:
call back URL should be the one which is configured

Reply:
Acknowledged.